.github/workflows/dockertests.yml

```diff
@@ -75,10 +75,10 @@ jobs:
         chmod +x contrib/free_disk_space.sh
         ./contrib/free_disk_space.sh
     - name: Set up JDK
-      uses: actions/setup-java@v3
+      uses: actions/setup-java@v4
       with:
         distribution: 'zulu'
-        java-version: '19'
+        java-version: '20'
```

Comment:
21

Reply:
@prabhu 21 didn't work for java-sec-code in the reachables test.

Reply:
oh. atom would not work on < 21. I think I know how to make java-sec-code work. Let me fork and update the project.

Reply:
For now, remove mvn compile for java-sec-code. Reachables would still work based on the sbom.

Reply:
https://github.com/ngcloudsec/java-sec-code - This would work with java 21